commands/src/main/resources/OSGI-INF/blueprint/jclouds-commands.xml

```diff
@@ -566,7 +566,7 @@ limitations under the License.
 
     <bean id="osFamilyCompleter" class="org.jclouds.karaf.commands.compute.completer.OsFamilyCompleter"/>
 
-  <bean id="osgiScriptEngineManager" class="org.jclouds.karaf.commands.table.internal.OSGiScriptEngineManager">
+  <bean id="osgiScriptEngineManager" class="org.jclouds.karaf.commands.table.internal.osgi.OSGiScriptEngineManager">
```

Comment:
Er...oops :-( Thanks for catching that! Mind changing the commit title to refer to the JIRA issue?

Thanks, @nacx!